phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/ui/FailurePanel.java

```diff
@@ -59,7 +59,7 @@ public FailurePanel() {
     public void setFailure(String notification, Optional<Exception> reason) {
         this.failureNotification.setText(notification);
         if (reason.isPresent()) {
-            this.failureReason.setText(tr("Reason: ") + reason.get().getLocalizedMessage());
+            this.failureReason.setText(tr("Reason:") + " " + reason.get().getLocalizedMessage());
```

Comment:
Probably better: `"Reason: {0}"`

Reply:
Does this become:
`this.failureReason.setText(tr("Reason: {0}"), reason.get().getLocalizedMessage());`

Reply:
done